pkgs/tools/security/age/default.nix

```
@@ -53,6 +53,11 @@ buildGoModule rec {
     fi
   '';
 
+  # plugin test is flaky on darwin, see https://github.com/FiloSottile/age/issues/517
+  preCheck = lib.optionalString stdenv.isDarwin ''
+    buildFlagsArray+=("-skip" "TestScript/plugin")
+  '';
```

```suggestion
  checkFlags = lib.optionals stdenv.isDarwin [
    "-skip" "TestScript/plugin"
  ];
```

i think using checkFlags here works.